notebooks/3D-pose-estimation-webcam/3D-pose-estimation.ipynb

```diff
@@ -31,7 +31,33 @@
     "    - [Draw 2D Pose Overlays](#Draw-2D-Pose-Overlays)\n",
     "    - [Main Processing Function](#Main-Processing-Function)\n",
     "- [Run](#Run)\n",
-    "\n"
+    "\n",
+    "\n",
+    "### Installation Instructions\n",
+    "\n",
+    "This is a self-contained example that relies solely on its own code.\n",
+    "\n",
+    "We recommend  running the notebook in a virtual environment. You only need a Jupyter server to start.\n",
+    "For details, please refer to [Installation Guide](../../README.md).\n",
```

Comment:
Do we need to provide link to GitHub instead of relative file path?

Reply:
the main scenario when this instruction used, when user download single notebook instead of cloning repo, so I think it is better to use github link

Reply:
Done